docs/docs/path_algorithm.md

```diff
@@ -0,0 +1,406 @@
+---
+title: "Path algorithms"
+linkTitle: "Path algorithms"
```

Comment:
Regarding `docs/docs/images/road_network.png`
can we generate the graph using RedisInsight and overlay the missing data on top of it, for consistency with the other images ?